index.html

```diff
@@ -4,15 +4,142 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>upcoming movie by genre</title>
```

Comment:
Hey, I think this line 7 is not necessary, could you delete it?

Reply:
i think i fixed it